server/models/service-process.js

```diff
@@ -0,0 +1,132 @@
+var async = require('async');
+var debug = require('debug')('strong-mesh-models:server:service-process');
+
+module.exports = function(ServiceProcess) {
+  function recordFork(instanceId, pInfo, callback) {
+    debug('Process forked: worker id %d pid %d ppid %d',
+      pInfo.id, pInfo.pid, pInfo.ppid);
+
+    var proc = new ServiceProcess({
+      pid: pInfo.pid,
+      parentPid: pInfo.ppid,
+      workerId: pInfo.id,
+      serviceInstanceId: instanceId,
+      startTime: new Date()
```

Comment:
It does, but currently supervisor does not return the actual start time in the event. If its required to report actual time, that should be a separate enhancement req/issue.